### PR TITLE
Added AdminInterface::preValidate method.

### PR DESCRIPTION
**Pull request by Koc**
Follow up of #3493

### CODE REVIEW

**Comment by OskarStark** [comment]
I would remove the `[3.0]` Prefix in the title of this PR @core23 and we need to move the Milestone from `3.0` to `4.0`, am i right?

**Comment by Soullivaneuh** [comment]
Please fill your PR of missing parts indicated by the "Needs" labels.

Feel free to ask if you want some precision of what is needed.

**Comment by Koc** [comment]
closed by #3857

File: Controller/CRUDController.php [hunk 1]
```diff
@@ -405,10 +405,7 @@ public function editAction($id = null, Request $request = null)
         $form->handleRequest($request);
 
         if ($form->isSubmitted()) {
-            //TODO: remove this check for 3.0
```
Comment (core23) [inline Controller/CRUDController.php]: This hack is still in the `master` branch.

**Comment by core23** [comment]
Not completly merged @Koc

File: Admin/AdminInterface.php [hunk 2]
```diff
@@ -655,11 +655,10 @@ public function create($object);
      */
     public function delete($object);
 
-//TODO: uncomment this method for 3.0
-//    /**
-//     * @param mixed $object
-//     */
-//    public function preValidate($object);
+    /**
+     * @param mixed $object
```
Comment (Soullivaneuh) [inline Admin/AdminInterface.php]: Should not be `object` instead of `mixed`?

**Comment by Soullivaneuh** [comment]
Oh, #3857 is merged now. Closing.

**Comment by Koc** [comment]
@Soullivaneuh no, see coment from @core23 above

**Comment by Soullivaneuh** [comment]
> Not completly merged @Koc

This one?

If yes, #3857 is merged.

Or I missing something. :-)

**Pull request closed without merging**